nibabel/streamlines/trk.py

```diff
@@ -388,13 +404,13 @@ def _read():
         return cls(tractogram, header=hdr)
 
     def save(self, fileobj):
-        """ Saves tractogram to a file-like object using TRK format.
+        """ Save tractogram to a filename or file-like object using TRK format.
```

Comment:
I thought the verb was supposed to be conjugated in third-person singular when describing what a function does?